api/maps/addMarker.tsx

```diff
@@ -0,0 +1,16 @@
+import { Marker } from '@react-google-maps/api';
+import { Project } from '../../types/helper';
+
+export default function addMarker(projects: Project[] | null) {
+  return projects?.map((project, index) => {
+    return project.approved ? (
+      <Marker
+        key={index}
```

Comment:
Not 100% sure why, but I think it's not recommended to use index as the key for map functions. We should probably use the project id since it uniquely identifies each project! [https://robinpokorny.com/blog/index-as-a-key-is-an-anti-pattern/](url)

Reply:
yes, please use **project_id** instead of **index**. This will also help with future sprints when we have to pull data for specific projects since we can use the unique project_id for this!

Reply:
fixed!